src/control/server/storage/config.go

```diff
@@ -43,6 +43,7 @@ const (
 	accelOptMoveName = "move"
 	accelOptCRCName  = "crc"
 
+	bdevRoleNoneName = "n/a"
```

Comment:
Could it make sense to have the role data instead ?

Reply:
As per conversation with @Michael-Hennecke , "roles" should only exist in MD-on-SSD mode, so no roles should be described in PMem mode.